docker-compose.yml

```diff
@@ -52,7 +52,6 @@ services:
     hostname: notesnook-db
     volumes:
       - dbdata:/data/db
-      - dbdata:/data/configdb

```

Comment:
Why?

Reply:
You mapped  `dbdata` to different directories `/data/db` and `/data/configdb`, The directory `/data/configdb` is not necessary, so I deleted it.

Reply:
when `/data/db` and `/data/configdb` containing the same file, may it cause conflicts?

Reply:
Config db is required and used internally by MongoDB: https://www.mongodb.com/docs/manual/reference/config-database/

Reply:
https://www.mongodb.com/docs/manual/reference/program/mongod/#std-option-mongod.--configsvr

> --configsvr
  Required if starting a config server.
  Declares that this mongod instance serves as the [config server](https://www.mongodb.com/docs/manual/core/sharded-cluster-config-servers/#std-label-sharded-cluster-config-server) of a sharded cluster. When running with this option, clients (i.e. other cluster components) cannot write data to any database other than config and admin. The default port for a mongod with this option is 27019 and the default [--dbpath](https://www.mongodb.com/docs/manual/reference/program/mongod/#std-option-mongod.--dbpath) directory is /data/configdb, unless specified.

The `/data/configdb` directory is only used when the `--configsvr` option is enabled In addition, there may be file conflicts when two different directories are mapped to one volume, so I deleted the second mapping. If you don't like this processing method, I can add `configdb` volume to map to `/data/configdb`